src/config/index.ts

```diff
@@ -4,8 +4,12 @@ export default {
     environment: process.env.NODE_ENV || 'development',
     defaultMaxAge: 86400,
     port: 4028,
+    clientApiName: 'firefox-api-proxy',
```

Comment:
nit: I think you meant to name it `clientName` instead of `clientApiName`?

Reply:
Very observant! You're right.